Localization/LocalizationDatabase.cpp

```diff
@@ -0,0 +1,147 @@
+#include "LocalizationDatabase.h"
+
+
+
+void commonItems::LocalizationDatabase::ScrapeLocalizations(const ModFilesystem& mod_filesystem)
+{
+	Log(LogLevel::Info) << "Reading Localization...";
+
+	const auto localization_files = mod_filesystem.GetAllFilesInFolderRecursive("localization");
```

Comment:
V2, EU4, HOI4 and CK2 have a folder named "localisation". Shouldn't matter, won't apply this retroactively, but having it tweakable could be useful for new stuff... maybe.